cpp/src/parquet/column_reader.h

```diff
@@ -115,11 +116,30 @@ class PARQUET_EXPORT PageReader {
                                           bool always_compressed = false,
                                           const CryptoContext* ctx = NULLPTR);
 
+  // If skip_page_callback_ is present (not null), NextPage() will call the
+  // callback function exactly once per page in the order the pages appear in
+  // the column. If the callback function returns true the page will be
+  // skipped. The callback will be called only if the page type is DATA_PAGE or
```

Comment:
I think this would read better if it was advertised as a page filter (returning true to include a page and false to exclude it).
Calling it a "skip page callback" may give the impression that it will only be called by the `Skip` APIs.

Reply:
Done.

Reply:
Ok, but this should then skip the page if the returned value is false, not true.

Reply:
I feel like since it is a filter, it should return true if it is filtering? But I am not opposed to the other way around, especially if there is a convention for how it should be.